src/Elastic.Apm/Report/PayloadSenderV2.cs

```diff
@@ -55,6 +55,7 @@ public PayloadSenderV2(IApmLogger logger, IConfigurationReader configurationRead
 			servicePoint.ConnectionLimit = 20;
 
 			_httpClient = new HttpClient(handler ?? new HttpClientHandler()) { BaseAddress = serverUrlBase };
+			_httpClient.DefaultRequestHeaders.UserAgent.Add(new ProductInfoHeaderValue(new ProductHeaderValue("elasticapm-dotnet", _service.Agent.Version)));
```

Comment:
I recommend to use `$"elasticapm-{Elastic.Apm.Consts.AgentName}"` instead of `"elasticapm-dotnet"`

Reply:
Ops, I didn't see this constant. Will change it

Reply:
fixed